app/components/home/sampleData.js

```diff
@@ -1,51 +1,87 @@
+// Create sample events that have dates that are in the past, present and future
+const nowDate = new Date();
+const futureDate = new Date();
+const pastDate = new Date();
+futureDate.setHours(nowDate.getHours() + 48);
+pastDate.setHours(nowDate.getHours() - 48);
+
 const sampleEvents = [
   {
     name: 'New Year Wild Party',
-    date: '01-01-2018',
+    dateObj: nowDate,
+    date: nowDate.toDateString(),
     venue: 'Adam\'s House',
-    description: 'Party all night at Pasir Ris!',
```

Comment:
`const nowDate = new Date();`

They say this will give me the doublequote. How to overcome this?